frontend/src/containers/House/House.js

```diff
@@ -9,13 +9,16 @@ class House extends Component {
   componentWillMount = () => {
     this.setState( {new_house: this.props.house});
   };
+
   componentWillReceiveProps = (nextProps) => {
     this.setState( { new_house: nextProps.house});
   };
-  houseFormSubmit = e => {
+
+  houseFormSubmit = e => { // eslint-disable-line no-unused-vars
```

Comment:
good catch. I would prefer to just leave the "e" out and have `= () => {` rather than disable rules? Any reason for keeping an unused param @vivekimsit ?

Reply:
I have no idea but you maybe interested in using the event object later? I thought not to touch your code.